src/main/java/net/daveyx0/primitivemobs/entity/monster/EntityBrainSlime.java

```diff
@@ -101,6 +101,11 @@ public IEntityLivingData onInitialSpawn(DifficultyInstance difficulty, @Nullable
     {
         int i = this.rand.nextInt(3);
```

Comment:
If size-0 aren't intended, a cleaner alternative might be:
`int i = this.rand.nextInt(2)+1; `